lib/onboarding.py

```diff
@@ -29,8 +24,20 @@ def start_onboarding():
         if res == sublime.DIALOG_YES:
             link_opener.open_browser_url(
                 'https://github.com/kiteco/KiteSublime/blob/master/README.md')
-        return res == sublime.DIALOG_NO
+        settings.set("should_onboard_"+lang, False)
```

Comment:
I think settings should be a bit more user friendly to read. So I would propose having 3 settings:

`show_python_help`: Controls whether or not the Python onboarding flow should run.
`show_javascript_help`: Controls whether or not the JS onboarding flow should run.
`show_go_help`: Controls whether or not the Go onboarding flow should run.

Also, I would get rid of the `show_go_beta_notif`, etc settings. As mentioned in my comment above, we should merge the beta notification logic into the onboarding logic. I don't see an added benefit to having 2 separate code paths for behavior that should go away soon.

Reply:
You'll also need to define these settings in the [KiteSublime.sublime-settings](https://github.com/kiteco/KiteSublime/blob/master/KiteSublime.sublime-settings) file.

Reply:
@its-dhung @plungg It seems that the suggestions here are for the user to be able to control and toggle at will whether they see onboarding for a particular language. I took another look at the lexical spec's section for [onboarding](https://kite.quip.com/HRLNAsgV9IYb/link-Product-Spec-Lexical-Completions#XGJACAnCFUY) (it seems that I misunderstood this the first time through with this initial PR) and the behavior outlined there would make it difficult for a user to toggle the settings flag and get expected onboarding behavior. What should be done?

Reply:
Patrick and I talked about it, and it's an implementation detail. I'll get this working as described.